dist/pathfora.js

```diff
@@ -1014,11 +1014,23 @@
       cb(window.lio.data);
       return;
     } else if (window.jstag && typeof window.jstag.getEntity === 'function') {
+
+      var version = parseInt(window.jstag.version, 10);
       // > jstag 3.0.0
-      var entity = window.jstag.getEntity();
-      if (entity.data && entity.data.user) {
-        cb(entity.data.user);
-        return;
+      if (version >= 3) {
+        window.jstag.send('entityReady', function () {
```

Comment:
I have never seen this format before. 🤔 could we not do:

`window.jstag.entityReady(entity => ...)`

Reply:
updated